GameServer/Logic/Packets/PacketHandlers/Handlers/HandleSpawn.cs

```diff
@@ -41,7 +41,23 @@ public bool HandlePacket(Peer peer, byte[] data)
             var bluePill = _itemManager.GetItemType(_game.GetMap().GetBluePillId());
             var itemInstance = peerInfo.GetChampion().Inventory.SetExtraItem(7, bluePill);
             var buyItem = new BuyItemAns(peerInfo.GetChampion(), itemInstance);
-            _game.PacketHandlerManager.sendPacket(peer, buyItem, Channel.CHL_S2C);
+            _game.PacketHandlerManager.sendPacket(peer, buyItem, Channel.CHL_S2C);
+
+            // Runes
+            if (peerInfo.GetChampion().Runes != null)
```

Comment:
This should never be null. If there's no runes passed, just initialize it as empty.